NOTICE

```diff
@@ -767,7 +767,7 @@ OpenSSL License
  * The implementation was written so as to conform with Netscapes SSL.
  *
  * This library is free for commercial and non-commercial use as long as
- * the following conditions are aheared to.  The following conditions
+ * the following conditions are adheared to.  The following conditions
```

Comment:
We should not be changing license text; the misspellings here are part of the original document. See https://docs.oracle.com/en/enterprise-manager/cloud-control/enterprise-manager-cloud-control/13.5/oemli/original-ssleay-license.html for another example.

Reply:
Reverted in 52dc3e8